frozen-abi/src/abi_example.rs

```diff
@@ -457,6 +451,13 @@ impl AbiExample for std::path::PathBuf {
     }
 }
 
+#[cfg(not(target_os = "solana"))]
+impl AbiExample for std::time::SystemTime {
+    fn example() -> Self {
+        std::time::SystemTime::now()
```

Comment:
I might be dumb. How will this produce a consistent digest?

Reply:
not dump. added explicit comment: cfaf370ba7f2dc1c81fa65c7e1ebb76f81ee32a1

Reply:
I think there's another way to create one safely: https://doc.rust-lang.org/std/time/struct.SystemTime.html#associatedconstant.UNIX_EPOCH

Which is also constant 😉

Reply:
oh, `UNIX_EPOCH`. nice. you're right: b564f05637ebc1ce32dad844d622c73f56bc9d00